Tetfuza/TetfuzaBackend/TetfuzaBackend.cs

```diff
@@ -356,11 +225,11 @@ private bool CheckMove(Coordinate center, FuzaPiece orientation)
                     char fuza = orientation.Piece[row][col];
                     int colCoord = center.xPos - (FuzaPiece.PIECE_SIZE / 2) + col;
                     int rowCoord = center.yPos - (FuzaPiece.PIECE_SIZE / 2) + row;
-                    if (((colCoord < 0 || colCoord >= BOARD_WIDTH
-                        || rowCoord < 0 || rowCoord >= BOARD_HEIGHT)
+                    if (((colCoord < 0 || colCoord >= TetfuzaBoard.BOARD_WIDTH
```

Comment:
You might be able to get rid of these checks if you make the board bigger than the display. Imagine an invisible border around the screen, so that pieces that run over the border just get drawn but the user only sees what’s inside the border

Reply:
That's a good point, however I think it might be more convoluted than it's worth, and might introduce unexpected errors if we decide to change the size of the board or pieces later on down the line.

Reply:
Just define the board size to be (whatever it is now)+1